Package.swift

```diff
@@ -27,6 +27,10 @@ let package = Package(
             name: "AccessibilitySnapshotCore",
             targets: ["AccessibilitySnapshotCore"]
         ),
+        .library(
+            name: "AccessibilitySnapshotParser",
```

Comment:
How much work is it going to be if we want to rename these things in the future?